ui/desktop/app/components/settings-card/index.hbs

```diff
@@ -0,0 +1,36 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<Hds::Card::Container @level='mid' @hasBorder={{true}} class='settings-card'>
```

Comment:
After reviewing figma designs, I notice:
- All the cards have at least header and body/content.
- The headers in all the cards, at least have icon + text. Some have status and/or actions.
- The bodies/content seems to be a grid with a variety of content: text, components, etc

I will suggest within <hds::card>
We have 2 divs: `.header`, `.body/content` and try to follow and reproduce this pattern within all the cards, this will facilitate re use and clarify for the future CSS.

Reply:
I have a header div block and yield the body/content from the consumers. Added a comment for more clarity